### DISCUSSION TITLE
Tips on creating pseudo-bulk replicates based on cell identity

### DISCUSSION THREAD

**Discussion by drosop** [stuart-lab/signac#1468] [Q&A] [answered]
Hi,

I want to create psuedobulk replicates from scATAC data for downstream analysis like Differential accessibility analysis, creating density heatmaps, motif enrichment. 

Can you please share suggestions on how to create pseudobulk replicates from signac preprocessed object or if I can export signac object and use it as an input to archR addGroupCoverages() function? 

I am a newbie in single cell ATAC analysis. 

Any kind of help is greatly appreciated.

Thank you,

**Comment by timoast** [comment]
What are pseudobulk replicates?

**Comment by drosop** [comment]
Creating pseudobulk (aggregating counts) based on cell-type for each sample. I want to do this for differential accessibility and/or differential motif analysis. Something similar to archR addGroupCoverages() function. [https://www.archrproject.com/bookdown/pseudo-bulk-replicates-in-archr.html]

I have created a multiome object using this [tutorial](https://stuartlab.org/signac/articles/pbmc_multiomic). 
If after annotating cell-type, I re-call peaks using group.by="cell_type"; Is it equivalent to pooling sparse data from individual cells of a cell-type into a pseudobulk sample to call peaks? 

Also, is there any existing way or any suggestion on if I can convert Signac Object to ArchR project?

Thank you so much for you help!

**Comment by timoast** [comment, marked as answer]
> Creating pseudobulk (aggregating counts) based on cell-type for each sample. I want to do this for differential accessibility and/or differential motif analysis. Something similar to archR addGroupCoverages() function.

We don't have any function for pseudoreplication using single-cell data. Bear in mind that if you do this, the pseudoreplicates will likely be much more similar to each other than real replicates and so your resulting p-values from a statistical test that assumes observations are independent will be artificially low.

If you have multiple real replicates, you can aggregate single-cell data together for each replicate separately and use that as input to a differential test using the `AverageExpression` or `AggregateExpression` functions

> If after annotating cell-type, I re-call peaks using group.by="cell_type"; Is it equivalent to pooling sparse data from individual cells of a cell-type into a pseudobulk sample to call peaks?

Running `CallPeaks` with a `group.by` variable will call peaks for each group of cells separately and combine the peak coodinates (as stated in the [documentation](https://stuartlab.org/signac/reference/callpeaks))

> Also, is there any existing way or any suggestion on if I can convert Signac Object to ArchR project?

No, ArchR uses a custom format that isn't cross-compatible with any other single-cell formats
